resources/js/views/Statistics.vue

```diff
@@ -16,145 +16,43 @@
-		<Card v-if="total !== undefined">
-			<template #content>
-				<div class="flex flex-wrap">
-					<span class="w-full font-bold text-xl">{{ "Total" }}</span>
-					<span class="w-20 text-muted-color-emphasis">{{ "Photos" }}:</span>
-					<span class="w-[calc(100%-5rem)] font-bold">{{ total.num_photos }}</span>
-					<span class="w-20 text-muted-color-emphasis">{{ "Albums" }}:</span>
-					<span class="w-[calc(100%-5rem)] font-bold">{{ total.num_albums }}</span>
-					<span class="w-20 text-muted-color-emphasis">{{ "Size" }}:</span>
-					<span class="w-[calc(100%-5rem)] font-bold">{{ sizeToUnit(total.size) }}</span>
-				</div>
-			</template>
-		</Card>
-		<div class="py-4" v-if="albumData !== undefined && albumData.length > 0">
+		<TotalCard v-if="total" :total="total" />
+		<div class="py-4" v-if="load && total">
 			<ToggleSwitch v-model="is_collapsed" class="text-sm"></ToggleSwitch> {{ "Collapse albums sizes" }}
 		</div>
-		<DataTable
-			:value="albumData"
-			size="small"
-			scrollable
-			scrollHeight="600px"
-			:loading="albumData === undefined"
-			v-if="albumData === undefined || albumData.length > 0"
-		>
-			<Column field="username" header="Owner" class="w-32"> </Column>
-			<Column field="title" sortable header="Title"></Column>
-			<Column field="num_photos" sortable header="Photos" class="w-16"></Column>
-			<Column field="num_descendants" sortable header="Children" class="w-16"></Column>
-			<Column field="size" header="Size" sortable class="w-32">
-				<template #body="slotProps">{{ sizeToUnit(slotProps.data.size) }}</template>
-			</Column>
-		</DataTable>
+		<AlbumsTable v-if="load" v-show="!is_collapsed" :show-username="true" :is-total="false" :album-id="undefined" @total="total = $event" />
+		<AlbumsTable v-if="load" v-show="is_collapsed" :show-username="true" :is-total="true" :album-id="undefined" />
 	</Panel>
 </template>
 <script setup lang="ts">
 import Button from "primevue/button";
 import Toolbar from "primevue/toolbar";
 import Panel from "primevue/panel";
-import StatisticsService from "@/services/statistics-service";
 import { useAuthStore } from "@/stores/Auth";
-import { computed, ref } from "vue";
+import { ref } from "vue";
 import { useLycheeStateStore } from "@/stores/LycheeState";
 import { storeToRefs } from "pinia";
 import { useRouter } from "vue-router";
-import { sizeToUnit, sizeVariantToColour } from "@/utils/StatsSizeVariantToColours";
-import MeterGroup from "primevue/metergroup";
-import Card from "primevue/card";
-import Column from "primevue/column";
-import DataTable from "primevue/datatable";
 import ToggleSwitch from "primevue/toggleswitch";
 import { onKeyStroke } from "@vueuse/core";
 import { shouldIgnoreKeystroke } from "@/utils/keybindings-utils";
-import { usePreviewData } from "@/composables/preview/getPreviewInfo";
-import SETag from "@/components/icons/SETag.vue";
-
-type TotalAlbum = {
-	num_photos: number;
-	num_albums: number;
-	size: number;
-};
-
-type SizeVairantData = {
```

Comment:
It's no longer there, but I'm annoyed I missed this typo.

Reply:
[4903b69](https://github.com/LycheeOrg/Lychee/pull/2581/commits/4903b696db71e0b59f4f076bab6b638e1a2b1d45)